db/migrate/20240718063323_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
```

Comment:
What is the use of user table ?

Reply:
there is no need of user it happend by mistake, when i was creating hr model.